README.md

```diff
@@ -1,15 +1,91 @@
-# Unified Distributed Execution
+<p align="center"><a href="https://unidist.readthedocs.io"><img width=77% alt="" src="https://github.com/modin-project/unidist/blob/master/docs/img/unidist_logo.png?raw=true"></a></p>
+<h2 align="center">Unified Distributed Execution</h2>
 
-The framework supports multiple execution backends: Ray, Dask, MPI and MultiProcessing.
+<p align="center">
+<a href="https://github.com/modin-project/unidist/actions"><img src="https://github.com/modin-project/unidist/workflows/master/badge.svg" align="center"></a>
+<a href="https://unidist.readthedocs.io/en/latest/?badge=latest"><img alt="" src="https://readthedocs.org/projects/unidist/badge/?version=latest" align="center"></a>
+</p>
 
-To run tests you need to install ``pytest`` package:
+### What is unidist?
 
+unidist is a framework that is intended to provide the unified API for distributed execution by supporting various performant execution backends and utilizing all of your CPUs cores. At the moment the following backends
+are supported underneath the hood:
+
+* [Ray](https://docs.ray.io/en/master/index.html)
+* [MPI](https://www.mpi-forum.org/)
+* [Dask Distributed](https://distributed.dask.org/en/latest/)
+* [Python Multiprocessing](https://docs.python.org/3/library/multiprocessing.html)
+
+unidist is designed to work in a [task-based parallel](https://en.wikipedia.org/wiki/Task_parallelism) model.
+
+Also, the framework provides a sequential ``Python`` backend, that can be used for debugging.
+
+### Instalation
+
+Currently, unidist can be installed from sources using ``pip``:
+
+```bash
+$ pip install git+https://github.com/modin-project/unidist
 ```
-pip install pytest
+
+This will install unidist directly from the repo without you having to manually clone it! Please be aware that these changes have not made it into a release and may not be completely stable.
```

Comment:
```suggestion
This will install unidist directly from the repo without you having to manually clone it! Please be aware that the latest changes have not made it into a release and may not be completely stable.
```